### ISSUE TITLE
Unable to download DevTools for the current target.

### ISSUE THREAD

**Issue by Joel-o1** [microsoft/vscode-edge-devtools#878]
**Environment (please complete the following information):**
 - OS (*e.g.,* Windows 10):
 - Extension version (*e.g.,* 1.2.3):

**Describe the bug:**

**Repro steps:**

**Expected behavior:**

**Additional context:**

**Comment by chris-miner** [comment]
**Environment:**
 - OS: macOS 12.1
 - Extension version: v1.4.3

**Describe the bug:**
VSCode is unable to download dev tools for the current target

**Repro steps:**
1. created a project with a single html file.
2. open the html file
3. got to the run and debug view, click on run and debug
4. Open the browser dev tools using the little icon on the debug panel
5. an edge dev tools panel opens and then after a few seconds displays the error about unable ....

**Expected behavior:**
I thought the dev tools would be displayed

**Additional context:**
Based on source code, I guessed what url is used for downloading the tools.  I then tried downloading the tools with curl:
curl -v https://devtools.azureedge.net/serve_file/@fd65e6bbedc86a22f2393b7cd8d1585c54cada42/vscode_app.html

Also not working.
